LondonBritishLibrary/orient/BLorient545.xml

```diff
@@ -31,21 +32,225 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 132</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
-                        <summary/>
-                        
-                        
+<msItem xml:id="ms_i1">
+<locus from="3r"/>
+<title type="complete" xml:lang="en" ref="LIT1960Mashaf">Missal of the Ethiopian Church</title>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i2">
+<locus from="3r"/>
+<title type="complete" xml:lang="en">Daily prayer</title>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i2.1">
+<locus from="3r"/>
+<title type="complete" xml:lang="en" ref="LIT1716Kidanz">Morning prayer</title>
+<incipit>ኪዳን፡ ዘነግህ፡ ዘተናገረ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ እምድኅረ፡ ትንሣኤሁ፡ እሙታን። በረከቱ፡ ወሀብተ፡ ረድኤቱ፡ የሀሉ፡ ምስለ፡ ንጉሥነ፡ ዮሐንስ፡
+ወምስለ፡ ንግሥትነ፡ ሰብለ፡ ወንጌል፡ ለዓለመ፡ ዓለም፡ አሜን። ይብል፡ ካህን፡ ቅዱስ፡ እግዚአብሔር፡ ቅ'፡ ኃያል፡ ቅ'፡ ሕያው፡ ዘኢይመውት፡</incipit>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i2.2">
+<locus from="5v"/>
+<title type="complete" xml:lang="en" ref="LIT1717Kidanz">Evening prayer</title>
+<incipit>ኪዳን፡ ዘሰርክ፡ ይብል፡ ካህን። ጸጋሁ፡ ለእግዚአብሔር፡ የሀሉ፡ ምስለክሙ።</incipit>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i2.3">
+<locus from="6v"/>
+<title type="complete" xml:lang="en" ref="LIT4711Mastabqwe">Various intercessory prayers: for the sick, travellers,
+rain, the fruits of the earth, the waters of the rivers, the regnant king (<persName ref="PRS10375Yohanne">John</persName>), the dead,
+children, and the peace of the Church.</title>
+<note>The note on the margin: <foreign xml:lang="gez">ዝኒ፡ ጸሎት፡ ዘሐዋርያት፡ ዘተሠርዐ፡ በሲኖዶስ፡</foreign></note>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i2.4">
+<locus from="10r"/>
+<title type="complete" xml:lang="gez" ref="LIT4346Prayer">ሊጦን፡ or <foreign xml:lang="grc">Λιτανεία</foreign></title>
+<incipit xml:lang="gez">ወካዕበ፡ ናስተበቍዕ፡ ዘኵሎ፡ ይእኅዝ፡ እግዚአብሔር፡ አብ፡ ለእግዚአ፡ ወመድኃኒነ፡ ኢየሱስ፡ ክርስቶስ። እንዘ፡ ነአኵቶ፡ እስመ፡ ዓቀበነ፡ በኑኀ፡
+ሌሊት፡ ወአብጽሐነ፡ እምጽልመት።</incipit>
+<note>The note on the margin: ዝኒ፡ ጸሎት፡ ዘሊቃውንት።</note>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i2.5">
+<locus from="12v"/>
+<title type="complete" xml:lang="gez" ref="LIT1808Litonz">ሊጦን፡ ዘሰርክ፡ or <foreign xml:lang="grc">Λιτανεία</foreign> for Vespers</title>
+<incipit xml:lang="gez">አምላክነ፡ ዘዲበ፡ ኩራቤል፡ ይነብር።</incipit>
+<note>It comprises a prayer for the dead <foreign xml:lang="gez">ጸሎተ፡ ሙታን፡</foreign> on <locus target="#13r">f. 13r</locus>.</note>
+<note>The note on the margin of <locus target="#14v">f. 14v</locus>: <foreign xml:lang="gez">ዝየ፡ ተፈጸመ፡ ሊጦን።</foreign></note>
```

Comment:
All the same applicable here, additionally, please, add rather an msItem for the prayer for the dead neasted within this one.

Reply:
I divided into two msItems, as for the note on the margin
 see above.